### PR TITLE
Update Travis config for CI with Mongo 2.6

### PR DESCRIPTION
**Pull request by cchantep**

### CODE REVIEW

**Comment by cchantep** [comment]
Hi @fehmicansaglam , here is a PR inspired from your suggestion.

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,14 @@
 language: scala
-script: sbt ++$TRAVIS_SCALA_VERSION "testOnly * -- -exclude testCommands,mongo2_6 --"
+script: sbt ++$TRAVIS_SCALA_VERSION "testOnly * -- -exclude testCommands --"
 scala:
-  - "2.11.1"
+  - "2.11.3"
```
Comment (fehmicansaglam) [inline .travis.yml]: Why not `2.11.6`?

Comment (cchantep) [inline .travis.yml]: Why not. No strong opinion.

**Comment by cchantep** [comment]
@sgodbillon This one is passing the CI now

**Pull request merged**